src/Symfony/Component/HttpFoundation/Session/Storage/Handler/SessionHandlerFactory.php

```diff
@@ -48,7 +48,9 @@ public static function createHandler($connection): AbstractSessionHandler
             case !\is_string($connection):
                 throw new \InvalidArgumentException(sprintf('Unsupported Connection: "%s".', \get_class($connection)));
             case 0 === strpos($connection, 'file://'):
-                return new StrictSessionHandler(new NativeFileSessionHandler(substr($connection, 7)));
+                $savePath = substr('file://', 7);
```

Comment:
This looks like a typo to me, shouldn't it be the same as before?
```suggestion
                $savePath = substr($connection, 7);
```

Reply:
true, thanks for eagle eye 😉